site/content/en/docs/formats/mpii.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'MPII Human Pose Dataset'
+linkTitle: 'MPII Human Pose Dataset'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original MPII Human Pose Dataset is available
+[here](http://human-pose.mpi-inf.mpg.de/#overview).
+
+Supported annotation types:
+- `Bbox`
+- `Points`
+
+Supported attributes:
+- `center` (list)
```

Comment:
List of what? Need a more detailed description.

Reply:
Added a more detailed description.